src/components/layout/MunicipalityLayout.tsx

```diff
@@ -19,8 +20,18 @@ import { WithChildren } from 'types';
 
 import useMediaQuery from 'utils/useMediaQuery';
 
+import municipalities from 'data/gemeente_veiligheidsregio.json';
+
 export default MunicipalityLayout;
 
+type TMuncipality = {
+  name: string;
+  displayName?: string;
+  safetyRegion: string;
+  gemcode: string;
+  searchTerms?: string[];
+};
+
```

Comment:
```suggestion
export type TMunicipality = typeof municipalities;
```
This will automatically construct a type based on the JSON shape.